README.md

```diff
@@ -38,10 +42,115 @@ Keep original format (`png`) and set width to `200`:
 
 `http://cdn.example.com/static/buffalo.png?width=200`
 
-Resize to `200x300px` using `embed` method and change format to `webp`:
+Resize to `200px` using `embed` method and change format to `webp`:
+
+`http://cdn.example.com/static/buffalo.png?embed&format=webp&resize=200`
+
+<h2 align="center">API</h2>
+
+#### Resize
+
+##### Width
+
+`http://localhost:3000/buffalo.png?width=200`
```

Comment:
Maybe better if we use a consistent base like `cdn.example` (or updating others to be `localhost`)? And what do you think about using a markdown table? (since they are mostly one operation with remarks)

Reply:
For me `localhost` is easier to understand, as it will probably be the first way to interact with the package after running `npx ipx`, and it's easily mappable to another domain (dev => `http://localhost`, prod => `https://mydomain`) but it could be just me. 

The markdown table is a great idea, it will make the API docs more terse and digest, I'll make the modifications 👍